disk_areas.md

```diff
@@ -39,10 +41,13 @@ For more details you can look at: [Managing Data on Abel] and more generally at
 <img src="https://docs.google.com/drawings/d/e/2PACX-1vSY_KCj3fubTH1zk6ZkOL6eLhoOOuAbp4bfu1YkOAvkadHhPfbuZrsepwHCUpEqwr45Zqt2hlEoCwVk/pub?w=960&amp;h=720">
 
```

Comment:
I LOVE this figure, however, it a little bit mixes disk areas with compute nodes/slurm. Not sure how to sort that out yet. Maybe discuss on wednesday?

Reply:
ok - splitted